pace_neutrons_cli/utils.py

```diff
@@ -382,7 +382,7 @@ def install_MCR(interactive=False):
         print('This could take some time (15-30min)')
         print('------------------------------------')
         proc = subprocess.run(prefix + [installer_file, '-mode', 'silent', '-agreeToLicense', 'yes'],
-                              capture_output=True)
+                              capture_output=True, text=True)
         if proc.returncode != 0:
             print(proc.stderr.decode())
```

Comment:
The change above means you shouldn't need the `decode` here.

Reply:
So it would become this?

```
if proc.returncode != 0:
    print(proc.stderr)
```

Reply:
Yeah